extensions/cloudfiles-extensions/src/main/java/io/druid/storage/cloudfiles/CloudFilesAccountConfig.java

```diff
@@ -0,0 +1,33 @@
+package io.druid.storage.cloudfiles;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CloudFilesAccountConfig {
+
+	@JsonProperty
+	@NotNull
+	private String provider;
+
+	@JsonProperty
+	@NotNull
+	private String userName;
+
+	@JsonProperty
+	@NotNull
+	private String apiKey;
```

Comment:
(non-blocking) In general if we can avoid secrets or any authentication in plaintext stuff that would be awesome. Having this abstracted in a provider (see the aws credential provider) would be preferred if possible, but I do not think is grounds to block the PR since we have other places where auth secrets are in plain text.